hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -335,6 +335,14 @@
                     name: "OWNER_SW_CFG_ROM_EXT_BOOTSTRAP_EN",
                     size: "4"
                 }
+                {
+                    name: "OWNER_SW_CFG_ROM_SENSOR_CTRL_ALERT_CFG",
+                    size: "4" // 2 bit per alert (disabled, fatal) for (currently) 11 alerts -> 32 bit suffice
+                }
```

Comment:
@timothytrippel Do you agree that 2 OTP bit per alert (one to disable it, one to make it fatal) are suitable from a provisioning and ROM perspective?

Reply:
That suffices, but do we want these to be mubis? Since the underlying CSRs in SENSOR_CTRL that this info will get copied to, perhaps it does not matter (in #23152)?

Reply:
We do have integrity protection on the config bits in OTP (through the digest) and while getting sent over TL-UL (ECC), so I don't think they need to be MuBi in OTP.

Reply:
@vogelpi WDYT?

Reply:
I think 2 bits per alert should suffice as we also have the digest. If we store the OTP values as MuBis, this would require quite some bits on top. I don't know how much we have left. How do we store similar configuration values in OTP?

Reply:
Alert classification is stored as 8-bit sparse values in OTP (as is alert class enable and escalation policies): https://github.com/lowRISC/opentitan/blob/2aca3e991a8d36d46d48f886a868aa87fdf1e781/sw/device/silicon_creator/lib/drivers/alert.h#L19-L41

Reply:
Changed to MuBi4

Reply:
Thanks @andreaskurth , I think this makes sense.